app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesFragmentTest.kt

```diff
@@ -246,6 +276,58 @@ class PoliciesFragmentTest {
     }
   }
 
+  @Test
+  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
+    launch<PoliciesFragmentTestActivity>(
+      createPoliciesFragmentTestActivity(
+        getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { activityScenario ->
+      activityScenario.onActivity { activity ->
+
+        testCoroutineDispatchers.runCurrent()
+        val htmlParser = htmlParserFactory.create(
+          policyOppiaTagActionListener = mockPolicyOppiaTagActionListener,
+          displayLocale = appLanguageLocaleHandler.getDisplayLocale()
+        )
+        val textView: TextView =
+          activity.findViewById(R.id.policy_description_text_view)
+
+        val htmlResult: Spannable = htmlParser.parseOppiaHtml(
+          getResources().getString(R.string.terms_of_service_content),
+          textView,
+          supportsLinks = true,
+          supportsConceptCards = false
+        )
+        textView.text = htmlResult
+        // Verify the displayed text is correct & has a clickable span.
+        val clickableSpans = htmlResult.getSpansFromWholeString(ClickableSpan::class)
+        assertThat(clickableSpans).isNotEmpty()
+        // Call each of the spans.
+        clickableSpans[0].onClick(textView)
+        // Verify that the tag listener is called.
+        verify(mockPolicyOppiaTagActionListener).onPolicyPageLinkClicked(
```

Comment:
Only this should be needed--the intent verification below isn't needed if we're only verifying the mock.

Reply:
Removed.